datasets/ru/egrul/crawler.py

```diff
@@ -688,25 +693,34 @@ def parse_examples(context: Context):
             parse_xml(context, fh)
 
 
-def crawl_index(context: Context, url: str) -> Set[str]:
+def crawl_index_internal(context: Context) -> Set[str]:
     """
-    Crawl an index page with ZIP archives and return the URLs.
+    Crawl an index page with ZIP archives, fetch them, and return the local paths.
+
+    This function fetches the HTML index file using a relative path, parses it
+    to extract links to ZIP archives, and collects their relative paths.
+
     Args:
         context: The processing context.
-        url: The URL to crawl.
+
     Returns:
-        A set of ZIP archive URLs.
+        A set of relative paths to the ZIP archives listed in the index.
     """
+    local_index_path = context.get_resource_path("index.html")
+    fetch_internal_data(
```

Comment:
The list function for the internal bucket is now in `main`. We can use that to avoid having to parse the index HTML and just parse all the `.zip` in the folder :)